packages/chart/src/Pie.ts

```diff
@@ -424,6 +432,8 @@ export interface Pie {
     seriesPercentageFormat(): string;
     seriesPercentageFormat(_: string): this;
 }
+Pie.prototype.publish("showSeriesValue", false, "boolean", "Append data series percentage next to label");
+Pie.prototype.publish("seriesValueFormat", ",.0f", "string", "Number format used for formatting series values", null, { disable: w => !w.showSeriesValue() });
```

Comment:
What makes these two different to:
* showSeriesPercentage
* seriesPercentageFormat

Reply:
The "value" and the "percentage" can both be individually toggled.

Reply:
Can you update the description accordingly